CONTRIBUTING.md

```diff
@@ -102,7 +102,7 @@ a few things you can do to help out:
 - **Reviewing documentation changes.** Most documentation just needs a review
   for proper spelling and grammar. If you think a document can be improved in
   any way, feel free to hit the `edit` button at the top of the page. More info
-  on contributing to documentation here.
+  on contributing to documentation [here](#documentation).
```

Comment:
@vinckr can you move this to meta?

Reply:
yes, thanks for catching this @masoudtahmasebi93